dlc/src/lib.rs

```diff
@@ -417,7 +451,53 @@ pub fn make_funding_redeemscript(a: &PublicKey, b: &PublicKey) -> Script {
         .into_script()
 }
 
-/// Create an adaptor signature for the given cet
+fn combine_pubkeys(pubkeys: &Vec<PublicKey>) -> Result<PublicKey, Error> {
```

Comment:
isn't there a function on secp256k1's public API which takes in an array of pub keys?

Reply:
Not that I'm aware off, and checked quickly but couldn't find. Happy to hear if there is one though.

Reply:
https://github.com/bitcoin-core/secp256k1/blob/master/include/secp256k1.h#L760

Reply:
Oh thanks I missed that! Looks like in rust-secp256k1 they only use it to combine two pubkeys for now though: https://github.com/rust-bitcoin/rust-secp256k1/blob/master/src/key.rs#L369

I'll try to do a PR to add a version with multiple keys but it might take a bit time before it gets merged.